cobalt/devinfra/kokoro/build/linux/package.py

```diff
@@ -0,0 +1,24 @@
+#!/usr/bin/env python3
+"""Packages Cobalt for Linux."""
+
+import os
+import shutil
+import sys
+import tempfile
+
+out_dir = sys.argv[1]
```

Comment:
Script arguments should be self-documenting. Please use `argparse`, rather than taking the args directly from `sys.argsv`

Reply:
Agree with Kaido.

You can pretty much copy the argparse section from https://github.com/youtube/cobalt/blob/main/cobalt/build/archive_test_artifacts.py

Reply:
This follows the existing pattern which is
much more concise and I find more readable
than the code in `archive_test_artifacts.py`.
I prefer to keep  consistent with that pattern.